openapi.yaml

```diff
@@ -58,26 +54,30 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
-  /validate:
+  /parse:
     post:
-      summary: Validate the given AsyncAPI document.
+      summary: Parse the given AsyncAPI document.
       operationId: validate
       tags:
-        - validate
-        - validator
+        - parse
+        - parser
       externalDocs:
         name: Github Repository for the AsyncAPI Parser
         url: https://github.com/asyncapi/parser-js
       requestBody:
-        description: Validate the given AsyncAPI document with the AsyncAPI parser.
+        description: Parse the given AsyncAPI document.
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/ValidateRequest'
+              $ref: '#/components/schemas/ParseRequest'
       responses:
         "204":
```

Comment:
Maybe change to a `200` since we return JSON?

Reply:
You're right, I forgot about that 😄